src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -1076,6 +1078,10 @@ void program_node::init_onednn_primitive_attributes() {
                     post_ops.append_binary(dnnl::algorithm::binary_add, in_desc);
                     update_onednn_post_op_list(onednn_post_op_type::binary_add, dep_idx);
                 }
+            } else if (desc.typed_desc<eltwise>()->mode == eltwise_mode::sub) {
+                dnnl::memory::desc in_desc = onednn::layout_to_memory_desc(in);
+                post_ops.append_binary(dnnl::algorithm::binary_sub, in_desc);
+                update_onednn_post_op_list(onednn_post_op_type::binary_sub, dep_idx);
             } else {
```

Comment:
I think we are missing some assertion logic here. What if mode is not `sum`, nor `sub`?

Reply:
Added